### ISSUE TITLE
USWDS-Site - Guidance: Add additional guidance regarding custom styles + extending USWDS

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds-site#2562]
# Summary

Add guidance to outline how users can use custom styles and extend USWDS beyond what theme settings are available to them.

## Problem statement

Our current **[Getting started for developers - Part 3: Customization](https://designsystem.digital.gov/documentation/getting-started/developers/phase-three-customize/)** primarily focuses on using theme settings and design tokens to customize USWDS.

Our **[document page](https://designsystem.digital.gov/documentation/developers/#sass-and-theme-settings-2)** mentions using `_uswds-theme-custom-styles.scss` to add additional project CSS for customizing or adding to what USWDS provides but does not provide any examples or recommendations

## Proposed solution

Use some of the examples provided in the new USWDS-Tutorial - Part 2: Themeing, Customizing, and Extending to define how users can expect to add additional customizations to their projects.

This will help users to understand how they can customize the design system, and provide additional context on why they are expected to customize in this way.

**Comment by mejiaj** [comment]
Consider using the February monthly call help address this issue.

**Comment by mahoneycm** [comment]
## Now and later steps

We discussed breaking this out into now and later steps to quickly get something on the site and providing more robust guidance down the road.

### Now

We can link to the [USWDS-Tutorial](https://github.com/uswds/uswds-tutorial) which provides examples on theme settings, custom styles, and extending USWDS to create custom components. We could also link the monthly call video of James and I doing a walk through of the training.

### Later

We should establish our recommended pattern for users adding custom styles. Once that is established, we should update the [Phase 3: Customizing](https://designsystem.digital.gov/documentation/getting-started/developers/phase-three-customize/) and [documentation](https://designsystem.digital.gov/documentation/developers/) pages to provide additional information on how users can expect to customize USWDS without worry of breaking components or updates erasing their styles.

**Comment by mejiaj** [comment]
@mahoneycm the monthly call video already exists on https://designsystem.digital.gov/about/monthly-calls/. Are you saying we'd link to this in `Phase 3: Customizing`?

**Comment by mahoneycm** [comment]
@mejiaj  Yep! Personally, I think it'd be a good idea to make sure people know that the resource exists.
